### PR TITLE
367 add additional apply block logs

### PR DESCRIPTION
**Pull request by Katkevich**

### CODE REVIEW

File: libraries/chain/database/database.cpp [hunk 1]
```diff
@@ -961,8 +962,13 @@ inline void database::push_virtual_operation(const operation& op)
         FC_ASSERT(is_virtual_operation(op));
 
         auto note = create_notification(op);
+
+        operation_info ctx(op);
+        op_debug_log(ctx, "virt operation pre_apply BEGIN");
         notify_pre_apply_operation(note);
+        op_debug_log(ctx, "virt operation pre_apply END/post_apply BEGIN");
```
Comment (Andrei-Masilevich) [inline libraries/chain/database/database.cpp]: Separate. Use two log string instead

Comment (Andrei-Masilevich) [reply libraries/chain/database/database.cpp]: Instead BEGIN and END one string in 'notify' function starting would be enough

File: libraries/chain/database/database.cpp [hunk 2]
```diff
@@ -1652,9 +1671,16 @@ void database::apply_operation(const operation& op)
 {
     auto note = create_notification(op);
 
+    //    operation_info ctx(op);
```
Comment (abolinbsu) [inline libraries/chain/database/database.cpp]: Why are these logs are commented?

File: libraries/chain/database/database.cpp [hunk 3]
```diff
@@ -1652,9 +1671,16 @@ void database::apply_operation(const operation& op)
...
+    //    op_debug_log(ctx, "_pre_apply_operation BEGIN");
     notify_pre_apply_operation(note);
+    //    op_debug_log(ctx, "_pre_apply_operation END");
+
     _my->_evaluator_registry.get_evaluator(op).apply(op);
+
+    //    op_debug_log(ctx, "_post_apply_operation BEGIN");
```
Comment (Andrei-Masilevich) [inline libraries/chain/database/database.cpp]: Why commented?

**Pull request merged**